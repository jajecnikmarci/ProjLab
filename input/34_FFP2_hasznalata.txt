timercontrol enable
starttest
add room room1 3
add room room2 3 poisonous
add door room1 room2 door1
add item room1 FFP2 ffp2
add person room1 Student student1
pickup student1
printstate ffp2
gotoroom student1 room2
printstate room2
printstate student1
gotoroom student1 room1
endtimer timer2
printstate ffp2
printstate student1
gotoroom student1 room2
endtimer timer3
endtimer timer4
endtimer timer5
endtimer timer6
printstate student1
endtest