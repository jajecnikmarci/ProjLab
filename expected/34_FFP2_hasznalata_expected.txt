ffp2
	- immunityLength: 10
	- effect: 
		PoisonImmunity
room2
	- capacity: 3
	- people: 
		student1
	- doors: 
		door1
	- items: 
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: null
student1
	- location: room2
	- stunned: false
	- inventory: 
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		PoisonImmunity
	- souls: 3
ffp2
	- immunityLength: 8
	- effect: 
		PoisonImmunity
student1
	- location: room1
	- stunned: false
	- inventory: 
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		PoisonImmunity
	- souls: 3
student1
	- location: room2
	- stunned: true
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
